### PR TITLE
LLP-52-AdaptiveVersion

### PR DESCRIPTION
**Pull request by mdainys**

### CODE REVIEW

File: src/components/BurgerMenu.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect, useRef } from 'react';
+import Link from "next/link";
+import styles from "@/styles/BurgerMenu.module.scss" 
+
+function BurgerMenu() {
+ 
+
```
Comment (pinenetcoder) [inline src/components/BurgerMenu.js:7]: Extra spaces

File: src/styles/BurgerMenu.module.scss [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      overflow: hidden;
+      transform: translateX(100%);
+      transition: transform 0.3s ease-in-out;
+    }
+  
+    ul li {
+      padding: 20px;
+    }
+  
+
```
Comment (pinenetcoder) [inline src/styles/BurgerMenu.module.scss:41]: xtra spaces

File: src/styles/IndexHeroSlider.module.scss [hunk 3]
```diff
@@ -28,4 +28,149 @@
...
+        color: var(--main-text-color);  
+        max-width: 470px;
+        margin-bottom: 44px;
+      }
+    }
+  }
+}
+@media (max-width: 1024px) {
+  .indexPageSlider {
+    // padding: 0 15px 0 15px;
```
Comment (pinenetcoder) [inline src/styles/IndexHeroSlider.module.scss:94]: commented code

File: src/styles/IndexHeroSlider.module.scss [hunk 4]
```diff
@@ -28,4 +28,149 @@
...
+        color: var(--main-text-color);  
+        max-width: 470px;
+        margin-bottom: 44px;
+      }
+    }
+  }
+}
+@media (max-width: 768px) {
+  .indexPageSlider {
+    // padding: 0 15px 0 15px;
```
Comment (pinenetcoder) [inline src/styles/IndexHeroSlider.module.scss:136]: commented code

**Review by pinenetcoder — APPROVED** [review]

File: src/styles/IndexHeroSlider.module.scss (hunk 4 above)
Comment (mdainys) [reply src/styles/IndexHeroSlider.module.scss:136]: done

File: src/styles/IndexHeroSlider.module.scss (hunk 3 above)
Comment (mdainys) [reply src/styles/IndexHeroSlider.module.scss:94]: done

File: src/styles/BurgerMenu.module.scss (hunk 2 above)
Comment (mdainys) [reply src/styles/BurgerMenu.module.scss:41]: done

File: src/components/BurgerMenu.js (hunk 1 above)
Comment (mdainys) [reply src/components/BurgerMenu.js:7]: done

**Pull request merged**